libraries/shared/storage/diff_repository_test.go

```diff
@@ -214,190 +215,145 @@ var _ = Describe("Storage diffs repository", func() {
-			Expect(len(diffsOne)).To(Equal(1))
-			nextID := int(diffsOne[0].ID)
-			diffsTwo, errTwo := repo.GetNewDiffs(nextID, limit)
-			Expect(errTwo).NotTo(HaveOccurred())
-			Expect(len(diffsTwo)).To(Equal(1))
-			Expect(int(diffsTwo[0].ID) > nextID).To(BeTrue())
-		})
-	})
-
-	Describe("GetUnrecognizedDiffs", func() {
-		It("sends diffs that are marked as 'unrecognized'", func() {
-			fakePersistedDiff := types.PersistedDiff{
-				RawDiff:   fakeStorageDiff,
-				ID:        rand.Int63(),
-				EthNodeID: db.NodeID,
-				Status:    "unrecognized",
-			}
-			insertTestDiff(fakePersistedDiff, db)
-
-			diffs, err := repo.GetUnrecognizedDiffs(0, 1)
-
-			Expect(err).NotTo(HaveOccurred())
-			Expect(diffs).To(ConsistOf(fakePersistedDiff))
-		})
-
-		It("does not sends diffs that are marked as 'new'", func() {
-			unrecognizedPersistedDiff := types.PersistedDiff{
-				RawDiff:   fakeStorageDiff,
-				ID:        rand.Int63(),
-				Status:    "new",
-				EthNodeID: db.NodeID,
-			}
-			insertTestDiff(unrecognizedPersistedDiff, db)
-
-			diffs, err := repo.GetUnrecognizedDiffs(0, 1)
-
-			Expect(err).NotTo(HaveOccurred())
-			Expect(diffs).To(BeEmpty())
-		})
-
-		It("does not send diffs that are marked as 'transformed'", func() {
-			transformedPersistedDiff := types.PersistedDiff{
-				RawDiff:   fakeStorageDiff,
-				ID:        rand.Int63(),
-				Status:    "transformed",
-				EthNodeID: db.NodeID,
-			}
-			insertTestDiff(transformedPersistedDiff, db)
-
-			diffs, err := repo.GetUnrecognizedDiffs(0, 1)
-
-			Expect(err).NotTo(HaveOccurred())
-			Expect(diffs).To(BeEmpty())
-		})
-
-		It("does not send diffs that are marked as 'noncanonical'", func() {
-			noncanonicalPersistedDiff := types.PersistedDiff{
-				RawDiff:   fakeStorageDiff,
-				ID:        rand.Int63(),
-				Status:    "noncanonical",
-				EthNodeID: db.NodeID,
-			}
-			insertTestDiff(noncanonicalPersistedDiff, db)
-
-			diffs, err := repo.GetUnrecognizedDiffs(0, 1)
-
-			Expect(err).NotTo(HaveOccurred())
-			Expect(diffs).To(BeEmpty())
-		})
-
-		It("enables seeking diffs with greater ID", func() {
-			blockZero := rand.Int()
-			for i := 0; i < 2; i++ {
-				fakeRawDiff := types.RawDiff{
-					Address:      test_data.FakeAddress(),
-					BlockHash:    test_data.FakeHash(),
-					BlockHeight:  blockZero + i,
-					StorageKey:   test_data.FakeHash(),
-					StorageValue: test_data.FakeHash(),
+				insertTestDiff(fakePersistedDiff, db)
+
+				diffs, err := repo.GetPendingDiffs(0, 1)
+				Expect(err).NotTo(HaveOccurred())
+				if present {
+					Expect(diffs).To(ConsistOf(fakePersistedDiff))
+				} else {
+					Expect(diffs).To(BeEmpty())
 				}
-				persistedDiff := types.PersistedDiff{
-					RawDiff:   fakeRawDiff,
+			},
+			KeepStatus(storage.Pending),
+			SkipStatus(storage.New),
+			SkipStatus(storage.Noncanonical),
+			SkipStatus(storage.Transformed),
+			SkipStatus(storage.Unrecognized),
+			SkipStatus(storage.Unwatched),
+		)
+
+		DescribeTable("only unrecognized",
```

Comment:
minor nitpick but this description doesn't match the format of the two above

Reply:
Gah! That's cause I fixed the other two and missed this one :)